.kitchen.yml

```diff
@@ -56,3 +59,19 @@ suites:
   attributes:
     varnish:
       version: '4.0'
+      log_daemon: false
+      
+- name: distro_libraries
```

Comment:
Does 'distro' mean from a package manager? And 'vendor' mean install from source? Just wondering if we could make those more descriptive.